### PR TITLE
Url shortener using pyshorteners!

### PR DESCRIPTION
**Pull request by swatv3nub**

### CODE REVIEW

File: shorturl.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# from Freaky Userbot
+# Add pyshorteners in requirements.txt
+
+import pyshorteners
+
+from . import *
+
+@ultroid_cmd(pattern="shorturl ?(.*)")
```
Comment (ArnabXD) [inline shorturl.py:8]: pattern used shorturl

File: shorturl.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        )
+        await event.edit(short, parse_mode="HTML")
+    except Exception as e:
+        await event.edit("SomeThing Went Wrong. \nError : " + e)
+
+
+HELP.update(
+    {
+        "urlshortner": "UrlShortner\
+\n\nSyntax : .urlshort <link>\
```
Comment (ArnabXD) [inline shorturl.py:26]: u used the `shorturl` in pattern , does this `urlshort` works too ?

**Review by ArnabXD — CHANGES_REQUESTED** [review]
Fix Helper, mainly dynamic HNDLR
 And add `pyshorteners` in addon.txt of this repo

File: shorturl.py (hunk 2 above)
Comment (swatv3nub) [reply shorturl.py:26]: Aah wrong pattern in help😐

**Review by ArnabXD — APPROVED** [review]

**Comment by ArnabXD** [comment]
If someone plans to merge this then merge #36 too

**Pull request merged**